napari/settings/_napari_settings.py

```diff
@@ -19,7 +19,7 @@
 
 _CFG_PATH = os.getenv('NAPARI_CONFIG', _DEFAULT_CONFIG_PATH)
 
-CURRENT_SCHEMA_VERSION = Version(0, 6, 0)
```

Comment:
Since we changed a default value, we bump minor version as per the comment. I tested whether we need a migration function but it seemed to just work even when upgrading napari in a new env...